Lab3/snake.js

```diff
@@ -603,12 +605,7 @@ const game = {
 const settings = {
     isDebugModeOn : function() {
         // TODO: return true if the debug mode checkbox is checked, false otherwise
-        if (document.getElementById("debug-mode-on").checked = true){
-            return true;
-        }
-        else {
-            return false;
-        }
+        return document.getElementById("debug-mode-on").checked;
```

Comment:
In this case, the `.checked` property itself is a boolean value that is true if the checkbox is checked and false otherwise, so you can just return the property value directly